bin/test_runner.rb

```diff
@@ -129,12 +131,17 @@ def wait
   private
 
   def cloud_name(name)
+    # The rocket monkey naming convention goes like this:
+    # prefix_cloud_region_testname...
+    # What this test calls cloud name is actually 'cloud_region'
+    # So split on _ and get those two, for example.
+    # rl10lin_Google_Silicon_monitoring...
+    # yields 'Google_Silicon' as cloud name.
     name.split('_')[1..2].join('_')
   end
 
   def total_deps_up(filter)
-    deployments = @rsclient.deployments.index(filter: ["name==#{filter}"])
-    deployments.length
+    deployments = @rsclient.deployments.index(filter: ["name==#{filter}"]).length
```

Comment:
Initially I suggested using `size` instead of `length`.  It's personal preference, but `length` is often used in referencing the length of a string and `size` is used in referencing the number of items in an array type.  It makes it easier to read by allowing the assumption that the object using `length` is a string.  But, this is not customer facing, so no worries.